ci/run_ci.sh

```diff
@@ -28,38 +30,32 @@ diff_list=`git diff origin/master --name-only`
 # - anything in ci has been changed
 # - anything in utils has been changed
 # Else only run tests on directories that have changed
-if [[ `echo $diff_list | grep -v / | wc -l` -gt 0 || `echo $diff_list | grep ci/` || `echo $diff_list | grep utils/` ]]
-then
+if [[ `echo ${diff_list} | grep -cv /` -gt 0 || `echo ${diff_list} | grep -E "(ci|utils|image_resources)/"` ]]; then
   echo "Running full test"
-  test_list=`ls -d */ | grep -Ev "(utils|ci|ripsaw|image_resources)/"`
+  test_list=`find * -name ci_test.sh -type f -exec dirname {} \;`
```

Comment:
would -maxdepth 1 be appropriate here because the ci_test.sh has to appear in the top wrapper directory?   just being paranoid, but that matches what ls -d */ does more closely.  But overall this is an improvement, gets rid of grep -v.

Reply:
Adding -maxdepth flag... Thanks for the suggestion!